basic/main.py

```diff
@@ -128,11 +128,14 @@ def _train(config):
 
 
 def _test(config):
-    test_data = read_data(config, 'test', True)
+    assert config.out_name
```

Comment:
this just verifies that `config.out_name` is set, i should probably add a more informative error message